discovery/seata-discovery-nacos/src/main/java/io/seata/discovery/registry/nacos/NacosRegistryServiceImpl.java

```diff
@@ -129,6 +141,25 @@ public List<InetSocketAddress> lookup(String key) throws Exception {
         if (clusterName == null) {
             return null;
         }
+        Pattern nacosRegistryPatternForMeta = StringUtils.isBlank(getNamingProperties().getProperty(SERVICE_META_PATTERN))?
+                DEFAULT_CUSTOMER_REGISTRY_PATTERN : Pattern.compile(getNamingProperties().getProperty(SERVICE_META_PATTERN));
+        if(nacosRegistryPatternForMeta.matcher(getNamingProperties().getProperty(PRO_SERVER_ADDR_KEY)).matches()){
+            LOGGER.info("visit seata on aliyun with pubnet mode");
```

Comment:
The concept of `pubnet` needs to be rethought. The word `aliyun` should not appear.